tests/nnapi/specs/Ex/one_hot_ex_dynamic_nnfw.mod.py

```diff
@@ -27,9 +27,10 @@
 indice_shape = [2, 2]
 indice_input = [1, 2, 0, 2]
 
-depth = Int32Scalar("depth", 3)
-onvalue = Float32Scalar("on", 1.) # default value is 1.
-offvalue = Float32Scalar("off", 0.) # default value is 0.
+depth = Input("depth", "TENSOR_INT32", "{1}")
+onvalue = Input("onvalue", "TENSOR_FLOAT32", "{1}")
+offvalue = Input("offvalue", "TENSOR_FLOAT32", "{1}")
```

Comment:
@glistening It seems the off value is wrong. I think it should be 0.

Reply:
@periannath `{1}` is not the value of `offvalue`. It is the shape of `offvalue` tensor. You can find the value of `offvalue` in `offvalue_data`.

Reply:
@glistening Thanks, I didn't know that.